cms/check_used_space

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+# Copyright 2012-2020 CERN
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Authors:
+# - Donata Mielaikaite, <donata.mielaikaite@cern.ch>, 2020
+# - Fernando Garzon, <oscar.fernando.garzon.miguez@cern.ch>, 2020
+# - Eric Vaandering, <ewv@fnal.gov>, 2021
+# - Panos Paparrigopoulos, <panos.paparrigopoulos@cern.ch>, 2024
+
+"""
+Probe to check used space.
+"""
+
+import sys
+import traceback
+
+from rucio.core.rse import list_rses, get_rse_usage, list_rse_attributes, get_rse_limits
+from rucio.db.sqla import models
+from rucio.db.sqla.session import get_session
+
+from utils import common
+
+PrometheusPusher = common.PrometheusPusher
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+if __name__ == '__main__':
+    try:
+        session = get_session()
+        with PrometheusPusher() as manager:
+            for rse in list_rses():
+                sources = get_rse_usage(rse['id'])
+                attributes = list_rse_attributes(rse['id'])
+                country = attributes.get('country', 'UNKNOWN')
+                rse_type = session.query(models.RSE.rse_type).filter(models.RSE.id == rse['id']).scalar()
+                rse_type = str(rse_type).split('.', 1)[1]
+                limits = get_rse_limits(rse['id'])
+                rucio_used = None
+                static_used = None
+                free_space = None
+                prom_labels = {'rse': rse['rse'], 'country': country, 'rse_type': rse_type, 'source': ''}
+                label_names = ['rse', 'country', 'rse_type', 'source']
```

Comment:
Hmm. Ok, I don't understand that I guess. But maybe we can sort that out later if we get a chance

Reply:
Ah, the difference is shown here: https://github.com/ericvaandering/probes/blob/406cba6df688b60d093708798b3688ef9c7d61e5/cms/check_rse_files_rules#L83 This way the label names are specified in the parameterized string. Want to try it that way? In this case the statsd metric is explicitly defined. (Of course we are getting rid of the statsd)

Reply:
Ah, now I see, thanks Eric.

I don't fully understand how the whole ETL pipeline works but I feel adding all those labels in the metric name creates a bit of a mess. If statsd anyways adds those latter I can definitely alter the names and skip using this `labelnames` parameters.

Otherwise, this should be good to be merged. I ran it once in integration and it seems to be running fine, I was also able to browse the results from grafana.